api/src/main/java/org/openmrs/module/registrationcore/api/mpi/openempi/MatchedPatientFilter.java

```diff
@@ -0,0 +1,58 @@
+package org.openmrs.module.registrationcore.api.mpi.openempi;
+
+import org.openmrs.PatientIdentifier;
+import org.openmrs.api.APIException;
+import org.openmrs.module.registrationcore.api.mpi.common.MpiPatient;
+import org.openmrs.module.registrationcore.api.mpi.common.MpiProperties;
+import org.openmrs.module.registrationcore.api.search.PatientAndMatchQuality;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+
+import java.util.List;
+
+public class MatchedPatientFilter {
```

Comment:
It is a problem that this class is OpenEMPI-specific, but that we use it directly in the controller